src/app/shared/movies.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+
+export interface IMovie {
+  id: number
+  preview: string
+  title: string
+  genre: string
+}
+
+@Injectable()
+export class MoviesService {
+  getMovies(): Array<IMovie> {
+    return this.movies;
```

Comment:
This should return observable with Array<IMovie>

Reply:
done